src/commands/transaction/mosaicaddressrestriction.ts

```diff
@@ -0,0 +1,147 @@
+import chalk from 'chalk';
+import { command, ExpectedError, metadata, option} from 'clime';
+import {
+    Account,
+    Address,
+    Deadline,
+    MosaicAddressRestrictionTransaction,
+    MosaicId,
+    TransactionHttp,
+    UInt64,
+} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+import {MosaicRestrictionType} from '../../validators/mosaic.validator';
+
+export class CommandOptions extends ProfileOptions {
+    public static limitType = [0, 1, 2, 3, 4, 5, 6];
+
+    @option({
+        flag: 'p',
+        description: '(Optional) Select between your profiles, by providing a profile',
+    })
+    profile: string;
+
+    @option({
+        flag: 'f',
+        default: 0,
+        description: '(Optional) Maximum fee',
+    })
+    maxFee: number;
+
+    @option({
+        flag: 'i',
+        description: 'Identifier of the mosaic being restricted',
+    })
+    mosaicid: string;
+
+    @option({
+        flag: 'a',
+        description: 'Address being restricted',
+    })
+    targetAddress: string;
+
+    @option({
+        flag: 'k',
+        description: 'Restriction key relative to the reference mosaic identifier',
+    })
+    restrictionKey: string;
+
+    @option({
+        flag: 'v',
+        default: '0',
```

Comment:
FFFFFFFFFFFFFFFF

Reply:
I've resolved this problem as you say.

But I'm not sure this well be better, because SDK has already defined the parameter's default value 'FFFFFFFFFFFFFFFF'.

In SDK, if the parameter is undefined, SDK will treat this parameter as 'FFFFFFFFFFFFFFFF'.

@dgarcia360